### ISSUE TITLE
Billion laughs xml "bomb"

### ISSUE THREAD

**Issue by zach-taylor** [minimaxir/big-list-of-naughty-strings#58]
Example here: https://en.wikipedia.org/wiki/Billion_laughs

**Comment by minimaxir** [comment]
That seems more like a specific script than a set of strings.

**Comment by ssokolow** [comment]
It's part of a set of XML vulnerabilities which are very common because so many XML parsers don't include protection by default. (eg. the Python standard library is vulnerable and the docs say to use the `defusedxml` package from PyPI if you're parsing untrusted XML.)

https://docs.python.org/2/library/xml.html#xml-vulnerabilities

**Comment by zach-taylor** [comment]
Indeed! Microsoft Word (at least on Mac) crashes upon pasting the above example as plaintext.

**Comment by minimaxir** [comment]
A Word crash makes sense since the modern versions are XML-based.

Rethinking this.
